templates/index.html

```diff
@@ -71,6 +71,17 @@ <h1>List-Building Comparison</h1>
 								</label>
 								{% endif %}
 							</div>
+							<div class="col col1">
+								{% if k %}
+								<label class="placeholder"><span class="field_name">Number of articles for list</span>
+									<input type="text" value="{{k}}" placeholder="Placeholder text" id="k"/>
```

Comment:
I don't have strong feelings either way, but if you change the input type to be "number" instead of "text" here (and on Line 81), the input box will have those little arrows on it and be more obviously an integer that we're looking for (simple example: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_number).